static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -280,6 +280,14 @@ CStudioAuthoring.Module.requireModule(
 				content_style: rteStyleOverride,
 
 				setup: function (editor) {
+				  var addPadding = function () {
```

Comment:
Is it ok/intended that this will execute multiple times as you focus/blur the control?

Reply:
yes